Data/collections.json

```diff
@@ -5871,21 +5889,19 @@
                         "SpecificProduct":"Europe",
                         "Spatial": "Europe",
                         "Temporal":"2017",
```

Comment:
The comma at the end of the Temporal key value is causing the action failure, I guess